.github/workflows/ci.yml

```diff
@@ -127,6 +127,14 @@ jobs:
                     sed -i "/mpi4py/ d" requirements.txt
                 fi
                 build_py_project_in_conda_env
-                test_py_project
+
+                if test "$DOWNSTREAM_PROJECT" = "mirgecom"; then
+                    # Test main branch
+                    # Only test examples, not pytest tests, as those take too much time.
```

Comment:
Why not add a separate job for examples instead of not running mirgecom pytest at all?

Reply:
Imo, running the `pytest` suite for the downstream project is of little value.   It also takes a really long time (only 30 minutes for mirgecom@main, but pushing an hour+ once NS+AV tests are added).   I think making sure the examples run and get the correct answers is a sufficient amount of testing to claim the current development does not break mirgecom.

Reply:
OK, fair. You're on the receiving end of those potentially breaking changes, so if you say it's OK, I'm happy to trust you and take my shorter CI times. :)